test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -380,5 +384,5 @@ TEST_F(BlockQueryTest, GetTopBlockFail) {
   auto top_block_error = framework::expected::err(empty_blocks->getTopBlock());
   ASSERT_TRUE(top_block_error);
   ASSERT_EQ(top_block_error.value().error,
-            "error while fetching the last block");
+            "Failed to retrieve block with id 0");
```

Comment:
Maybe, it's better to `"Failed to retrieve block with id " + mock_file->last_id()`? Just to avoid numbers